app/src/main/java/com/simply/birthdayapp/commonpresentation/theme/Color.kt

```diff
@@ -22,3 +22,7 @@ val BottomNavActiveItemColor = Color(0xff963E5A)
 val BottomNavDisableItemColor = Color(0xffFFF5F4)
 
 val SearchBarHintTextColor = Color(0xff929292)
+val ShopIconBorderColor = Color(0xff3F3F3F)
+val ClickableTextColor = Color(0xFF383838)
+val FilledStarColor = Color(0xFFFFA726)
+val EmptyStarColor = Color(0xFF9B9B9A)
```

Comment:
Replace names that indicate where a color is used with names that describe the precise color.